### PR TITLE
Fix for historical replay of withdrawals

### LINKED ISSUE #199: When HISTORIC is enabled, remove 1-week search optimizations for BGP withdrawals
**Describe the bug**
It is not a bug for standard ARTEMIS; however, when someone does historical analysis, the withdrawal monitoring is not happening correctly.
Check lines
https://github.com/FORTH-ICS-INSPIRE/artemis/blob/master/backend/core/database.py#L1169
https://github.com/FORTH-ICS-INSPIRE/artemis/blob/master/backend/core/database.py#L1272
in core/database.py. The fix should be easy; if the env variable historic is enabled, then search through the entire db (deactivate these checks).

**Affected Component(s)**
- [X] Back-End (Database, Microservices, Containers, etc)
- [ ] Front-End (Flask, API, etc)
- [ ] Docs
- [ ] Build System

**Additional context**

### PR DESCRIPTION
**Pull request by vkotronis**
### Description of PR

What component(s) does this PR affect?

- [X] Back-End (Database, Microservices, Containers, etc)
- [ ] Front-End (Flask, API, etc)
- [ ] Docs
- [ ] Build System

Does the PR require changes on other components? If yes, please mark the components:

- [ ] Back-End (Database, Microservices, Containers, etc)
- [ ] Front-End (Flask, API, etc)
- [ ] Docs
- [ ] Build System

### Related Issue

Resolves #199

### Type

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Docs update
- [ ] None of the above

### Checklist:

- [X] I have read the **[contributing guide](https://github.com/FORTH-ICS-INSPIRE/artemis/blob/master/CONTRIBUTING.md)** and my code conforms to the guidelines.
- [ ] This change requires a change in the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by vkotronis** [comment]
Tested as follows:

1) Test input:
139.91.0.0/25|100|8|8 3 2 1 100|routeviews|tst.caida|A|"[{""asn"":555, ""value"":1}, {""asn"":555, ""value"":2}]"|1563580800
139.91.0.0/25|100|9|9 3 2 1 100|routeviews|tst.caida|A|"[{""asn"":555, ""value"":3}]"|1563580801
139.91.0.0/25||8||routeviews|tst.caida|W|"[{""asn"":555, ""value"":1}, {""asn"":555, ""value"":2}]"|1563580802
139.91.0.0/25||9||routeviews|tst.caida|W|"[{""asn"":555, ""value"":1}, {""asn"":555, ""value"":2}]"|1563580803
139.91.0.0/16|100|8|8 3 2 1 100|routeviews|tst.caida|A|"[{""asn"":555, ""value"":1}, {""asn"":555, ""value"":2}]"|1564358400
139.91.0.0/16|100|9|9 3 2 1 100|routeviews|tst.caida|A|"[{""asn"":555, ""value"":3}]"|1564358401
139.91.0.0/16||8||routeviews|tst.caida|W|"[{""asn"":555, ""value"":1}, {""asn"":555, ""value"":2}]"|1564358402
139.91.0.0/16||9||routeviews|tst.caida|W|"[{""asn"":555, ""value"":1}, {""asn"":555, ""value"":2}]"|1564358403

2) HISTORIC = false
The 1st /25 hijack does not come up at all.

3) HISTORIC = true
The 1st /25 hijack comes up correctly (withdrawn).

File: backend/core/database.py [hunk 1]
```diff
@@ -1160,13 +1161,17 @@ def _insert_bgp_updates(self):
...
+            )
+            timestamp_thres = datetime.datetime.fromtimestamp(timestamp_thres)
             query = (
                 "SELECT DISTINCT ON (hijacks.key) hijacks.peers_seen, hijacks.peers_withdrawn, "
                 "hijacks.key, hijacks.hijack_as, hijacks.type, bgp_updates.timestamp, hijacks.time_last "
                 "FROM hijacks LEFT JOIN bgp_updates ON (hijacks.key = ANY(bgp_updates.hijack_key)) "
                 "WHERE bgp_updates.prefix = %s "
                 "AND bgp_updates.type = 'A' "
-                "AND bgp_updates.timestamp >= NOW() - INTERVAL '1 WEEK' "
+                "AND bgp_updates.timestamp >= %s "
```
Comment (slowr) [inline backend/core/database.py]: we don't need the `1 WEEK` any more?

File: backend/core/taps/bgpstreamhist.py [hunk 2]
```diff
@@ -82,6 +83,7 @@ def parse_bgpstreamhist_csvs(prefixes_file=None, input_dir=None):
                                                     routing_key="update",
                                                     serializer="json",
                                                 )
+                                                time.sleep(0.1)
```
Comment (slowr) [inline backend/core/taps/bgpstreamhist.py]: why was this necessary?

Comment (vkotronis) [reply backend/core/taps/bgpstreamhist.py]: I just put it so that we do not just flood the detector(s) with 1000s or 100s of 1000s of pre-recorded messages at once.

Comment (vkotronis) [reply backend/core/taps/bgpstreamhist.py]: Do you think it is OK or does it hurt? I thought that since we replayed history proper operation is more imporant than speed (I suppose that without this mini rate-limiting ARTEMIS detectors and RMQ can be easily flooded)

File: backend/core/database.py (hunk 1 above)
Comment (vkotronis) [reply backend/core/database.py]: of course we do. Check line 1165.
